.devcontainer/devcontainer.json

```diff
@@ -1,19 +1,62 @@
 {
-    "dockerComposeFile": ["../development/docker-compose.yml", "../development/docker-compose.debug.yml"],
+    "name": "Nautobot Dev Container",
+    "dockerComposeFile": [
+        "../development/docker-compose.yml",
+        "../development/docker-compose.postgres.yml",
+        "../development/docker-compose.dev.yml"
+    ],
+    "features": {
+        "ghcr.io/devcontainers/features/docker-outside-of-docker:1": { }
+    },
     "service": "nautobot",
-	"workspaceFolder": "/source/",
-	"extensions": [
-		"ms-python.vscode-pylance",
-		"streetsidesoftware.code-spell-checker",
-		"eamodio.gitlens",
-		"oderwat.indent-rainbow",
-		"ms-python.python",
-		"ms-vsliveshare.vsliveshare"
-	],
-	"settings": {
-		"python.pythonPath": "/usr/local/bin/python",
-		"python.analysis.extraPaths": ["/source"],
-		"python.linting.pylintEnabled": true,
-		"python.linting.enabled": true
-	}
+    "shutdownAction": "stopCompose",
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "charliermarsh.ruff",
+                "DavidAnson.vscode-markdownlint",
+                "eamodio.gitlens",
+                "EditorConfig.EditorConfig",
+                "GitHub.vscode-pull-request-github",
+                "ms-python.python",
+                "ms-python.vscode-pylance",
+                "samuelcolvin.jinjahtml",
+                "tamasfe.even-better-toml"
```

Comment:
Just curious - should we add all of these extensions to `nautobot.code-workspace` as well?

Reply:
I'd be ok with it however I haven't used `DavidAnson.vscode-markdownlint`, `EditorConfig.EditorConfig`, `GitHub.vscode-pull-request-github`, `samuelcolvin.jinjahtml` or `tamasfe.even-better-toml`. I don't think `ms-python.vscode-pylance` is required in `nautobot.code-workspace` since `ms-python.python` installs it automatically. It may be necessary here because this is the list of extensions that will be automatically installed in the container. The `nautobot.code-workspace` is just a list of recommended extensions.